plugin/pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -357,6 +357,16 @@ func getResourceRequest(pod *api.Pod) resourceRequest {
 		result.memory += requests.Memory().Value()
 		result.milliCPU += requests.Cpu().MilliValue()
 	}
+	// take max_resource(sum_pod, any_init_container)
+	for _, container := range pod.Spec.InitContainers {
+		requests := container.Resources.Requests
+		if mem := requests.Memory().Value(); mem > result.memory {
```

Comment:
We get no benefit from that because result.Memory is int64

On Fri, May 6, 2016 at 12:25 PM, Derek Carr <notifications@github.com>
wrote:

> In plugin/pkg/scheduler/algorithm/predicates/predicates.go
> <https://github.com/kubernetes/kubernetes/pull/23567#discussion_r62353465>
> :
>
> > @@ -357,6 +357,16 @@ func getResourceRequest(pod *api.Pod) resourceRequest {
> >  		result.memory += requests.Memory().Value()
> >  		result.milliCPU += requests.Cpu().MilliValue()
> >  	}
> > +	// take max_resource(sum_pod, any_init_container)
> > +	for _, container := range pod.Spec.InitContainers {
> > +		requests := container.Resources.Requests
> > +		if mem := requests.Memory().Value(); mem > result.memory {
>
> use Cmp() on quantity
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/kubernetes/kubernetes/pull/23567/files/34568050edacec76ed78e1b16c3248f5cbdc81b3..805cb1e4b870ef2f22562afd5ecd559c82909659#r62353465>
>